.github/scripts/install-drgn.sh

```diff
@@ -10,5 +10,6 @@ sudo apt install bison flex libelf-dev libdw-dev libomp5 libomp-dev
 git clone https://github.com/osandov/drgn.git
 
 cd drgn
+git checkout -b drgn_february 0a6aaaae5d31ed142448f8220e208d65478ec80d
```

Comment:
Can we remove this line? If there is a problem with the github actions we fix the actions themselves. If there is a problem with drgn we fix drgn.